draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1968,11 +1958,21 @@ uCDN-A                         dCDN-B                         dCDN-C
 	Mandatory: Yes. The dCDN may use its own CDN PID if it does not want to expose the CDN PIDs of dCDNs.
 	</t>
 
-	</list>
-	</t>
+	</list></t>
+	</list></t>
```

Comment:
I'm counting the tags here, and it appears that you may have introduced an extra </t> here as part of tag cleanup, but I'm not 100% sure.  The author tools may indicate a problem if that is the case.

Reply:
I am running idnits before each commit, don't think there's an issue